### DISCUSSION TITLE
Server Groups

### DISCUSSION THREAD

**Discussion by Nightmarecaster** [stoatchat/discussions#509] [Feature Suggestions]
Like in Guilded, they have server groups that you can make within the server. That you can lock to a role you make in the server.

**Comment by sharkaccino** [comment]
I feel like it'd be easier to just implement sub-categories instead (and maybe the ability to give categories their own icons)

"Servers within servers" as an organizational concept is interesting. I'm only familiar with Guilded's approach to the idea, but their implementation ends up making their already confusing UI even more infuriating to navigate and comprehend. For what it tries to accomplish, I feel like sub-categories would solve most of the same issues without making the UI an absolute nightmare to use.
